systems/plants/RigidBodyManipulator.cpp

```diff
@@ -2967,6 +2967,8 @@ template DLLEXPORT_RBM void RigidBodyManipulator::doKinematics(MatrixBase<Vector
 template DLLEXPORT_RBM void RigidBodyManipulator::doKinematics(MatrixBase< Map<VectorXd> >  &, bool);
 template DLLEXPORT_RBM void RigidBodyManipulator::doKinematicsNew(const MatrixBase< Map<VectorXd> > &, const MatrixBase< Map<VectorXd> > &, bool, bool);
 template DLLEXPORT_RBM void RigidBodyManipulator::doKinematicsNew(const MatrixBase<VectorXd> &, const MatrixBase<VectorXd> &, bool, bool);
+template DLLEXPORT_RBM void RigidBodyManipulator::doKinematicsNew(const MatrixBase<Block<MatrixXd, -1, 1, true> > &, const MatrixBase<Block<MatrixXd, -1, 1, true> > &, bool, bool);
```

Comment:
We could at least get rid of the matGradMult explicit instantiations by moving the definition into the .h file, but I think I know what you're going to say about that.